### PR TITLE
Update docs Guides section

### PR DESCRIPTION
**Pull request by MinaPecheux**
Update the guides sections with:

- some step-by-step tutorials
- handy Lua snippets for easier templating

### CODE REVIEW

File: lua-docs/content/guides/tutorials/tutoworld.yml [hunk 1]
```diff
@@ -0,0 +1,1036 @@
...
+        - `claire.desert`
+        - `claire.market`
+
+        But, of course, you can also create your own! To do so, just go to the Item editor and assemble voxels to your liking as explained in the [Beginner's Guide](https://particubes.com/beginners-guide) :)
+
+        <i>Note: at this time, there isn't a dedicated map editor yet. So you simply make a map "item", and then import it as the map of your world using the [`Config`](/reference/config) object.</i>
+
+        For example, in my case, I prepared a small map called `minadune.tutoworld_map`:
+
+        // gif tutoworld map turnaround
```
Comment (gdevillele) [inline lua-docs/content/guides/tutorials/tutoworld.yml:120]: do you want to add a GIF here?

**Review by gdevillele — COMMENTED** [review]
it's really great! Thanks. 🤩
I only noticed a comment about a GIF, do you have the GIF file handy?

File: lua-docs/content/guides/tutorials/tutoworld.yml (hunk 1 above)
Comment (MinaPecheux) [reply lua-docs/content/guides/tutorials/tutoworld.yml:120]: Oups, nope - I forgot to add my .mp4 file, actually! ^^ (first thought I'd use GIFs but the engine was already made for movies and it's easier to prepare :)

Thanks, nice catch - I'll fix it!

Comment (gdevillele) [reply lua-docs/content/guides/tutorials/tutoworld.yml]: great ! I remember now, we chose video instead of GIF for the compression (smaller files)

**Comment by gdevillele** [comment]
@MinaPecheux it looks good to me ! I updated a text in the last commit, feel free to merge if you feel it's all good. We'll fix things afterwards if necessary, and it will make @RacineC 's review easier.

**Pull request merged**